ts/features/itwallet/presentation/components/ItwPresentationClaimsSection.tsx

```diff
@@ -37,14 +38,11 @@ export const ItwCredentialClaimsSection = ({
         <H6 color="grey-700">{title}</H6>
         {canHideValues && renderHideValuesToggle()}
       </View>
-      <View>
-        {claims.map((claim, index) => (
-          <React.Fragment key={claim.id}>
-            {index !== 0 && <Divider />}
-            <ItwCredentialClaim claim={claim} hidden={valuesHidden} />
-          </React.Fragment>
-        ))}
-      </View>
+      <ItwQrCodeClaimImage
+        claim={data.parsedCredential.link_qr_code}
+        valuesHidden={valuesHidden}
+      />
+      <ItwCredentialClaimsList data={data} />
```

Comment:
Did you forget to pass the `isHidden` prop? It's no longer possible to hide claims.

Reply:
Nice catch! Thank you
Addressed here https://github.com/pagopa/io-app/pull/6074/commits/6bc44033749b9f19684ad3e8882b7f315230066c